hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/impl/TestAwsClientConfig.java

```diff
@@ -18,9 +18,11 @@
 
 package org.apache.hadoop.fs.s3a.impl;
 
+import java.io.IOException;
 import java.time.Duration;
 import java.util.Arrays;
 
+import org.apache.hadoop.util.Lists;
```

Comment:
nit: move to the org.apache.* section (we aren't as strict as spark and leave existing files mostly alone to avoid cherrypicking, but we try...)

Reply:
>however, as per testing.md, I do need to know which s3 region/endpoint you tested against, even if it is some minio instance on a raspberry pi. Its our way of ensuring you've done the integration tests. once merged the rest of us get to run those same tests in our configs, we quite often find the odd quirk; i'll add a test header to include coverage.

I have added us-west-2 as bucket region where I ran integ test in my PR description. 

Below is my auth-keys.xml
```
<configuration>
    <property>
        <name>test.fs.s3a.name</name>
        <value>**redacted**</value>
    </property>
    <property>
        <name>fs.contract.test.fs.s3a</name>
        <value>${test.fs.s3a.name}</value>
    </property>
    <property>
        <name>fs.s3a.access.key</name>
        <description>AWS access key ID. Omit for IAM role-based authentication.</description>
        <value>**redacted**</value>
    </property>
    <property>
        <name>fs.s3a.secret.key</name>
        <description>AWS secret key. Omit for IAM role-based authentication.</description>
        <value>**redacted**</value>
    </property>
    <property>
        <name>test.sts.endpoint</name>
        <description>Specific endpoint to use for STS requests.</description>
        <value>sts.amazonaws.com</value>
    </property>
    <property>
        <name>fs.s3a.session.token</name>
        <value>**redacted**</value>
    </property>
    <property>
        <name>test.fs.s3a.sts.enabled</name>
        <value>false</value>
    </property>
    <property>
        <name>fs.s3a.endpoint</name>
        <value>s3.us-west-2.amazonaws.com</value>
    </property>
</configuration>
```